Makefile

```diff
@@ -3,15 +3,15 @@ ci: test
 	npm run docs
 
 test: lint
-	npm run test
+	npm run $@
 
 lint: node_modules
-	npm run lint
+	npm run $@
 
 clean:
 	rm -rf node_modules
 
 node_modules: package.json
 	npm install
 
-.PHONY: node_modules
+.PHONY: clean
```

Comment:
@danbev you're right, I didn't realize that it would ignore prerequisites - my mistake.